db/migrate/20150826192731_add_execution_to_interface.rb

```diff
@@ -0,0 +1,24 @@
+class FakeNic < ActiveRecord::Base
+  self.table_name = 'nics'
+
+  def type
+    Nic::Managed
+  end
+end
+
+class AddExecutionToInterface < ActiveRecord::Migration
+  def up
+    add_column :nics, :execution, :boolean, :default => false
+    add_column :hosts, :execution_interface, :integer
```

Comment:
missing in down, but I don't think this column is used, or at least is covered by method of same name